pkg/controller/jobframework/reconciler.go

```diff
@@ -363,6 +362,15 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, nil
 	}
 
+	if !*wl.Spec.Active {
```

Comment:
I'm not sure what will happen when we receive a Workload object that was created before the `Active` field was added.

Could this pointer be nil? I guess we can just call `!ptr.Deref(wl.Spec.Active, true)` to be on the safe side.